zebra-consensus/src/transaction/tests.rs

```diff
@@ -397,6 +441,55 @@ async fn v5_transaction_with_transparent_transfer_is_accepted() {
     assert_eq!(result, Ok(transaction_hash));
 }
 
+/// Test if V5 coinbase transaction is accepted.
+#[tokio::test]
+// TODO: Remove `should_panic` once the NU5 activation heights for testnet and mainnet have been
+// defined.
+#[should_panic]
```

Comment:
Minor: Maybe also add this to #1841?

Reply:
I just noticed a bug here: these functions should all use `Testnet`, because it will have an activation height first.

(Ideally they should use both networks, but that's a larger change.)

Reply:
Bug fixed in 09b815cf, also I updated the ticket, and added "any other should_panic" tests.